astronomer/providers/core/example_dags/example_file_sensor.py

```diff
@@ -24,10 +26,30 @@
     default_args=default_args,
     tags=["example", "async", "core"],
 ) as dag:
+    start = EmptyOperator(task_id="start")
+
+    create_file = BashOperator(
+        task_id="create_file",
+        bash_command="sleep 10 && touch /usr/local/airflow/dags/example_file_async_sensor.txt",
+    )
+
     # [START howto_sensor_filesystem_async]
     file_sensor_task = FileSensorAsync(
         task_id="file_sensor_task",
-        filepath="example_file_async_sensor.txt",
+        filepath="/usr/local/airflow/dags/example_file_async_sensor.txt",
```

Comment:
```suggestion
        filepath="/tmp/example_file_async_sensor.txt",
```
If the above suggestion can be considered.